### DISCUSSION TITLE
The initial admin user/pass is not mentioned in docker self-hosted guide

### DISCUSSION THREAD

**Discussion by parsa-asgari** [chatwoot/chatwoot#9191] [Self hosted]
Hi,
According to this link: [ChatWoot Docker docs](https://www.chatwoot.com/docs/self-hosted/deployment/docker), one could install chatwoot as a docker-compose deployment. However, the admin user/pass is not mentioned. 

It would be great if you could mention the admin credentials.

**Comment by vishnu-narayanan** [comment]
For a production deployment, no user is created. You will be shown on boarding screen and you will need to create the first user.

**Reply by parsa-asgari** [reply]
Thanks For the answer.
Thing is, sometimes the boarding screen does not show up as expected.

**Reply by parsa-asgari** [reply]
Well, I think that in case of the docker deployment, one should run the migrations command: 
`docker compose run --rm rails bundle exec rails db:chatwoot_prepare`
And then, run the `docker compose up -d` immediately.

It would be great if you could reflect this in the documentation.

**Reply by vishnu-narayanan** [reply]
@parsa-asgari Step 4 of the docker deployment docs mentions the DB migration part. Which guide did you follow?

ref: https://www.chatwoot.com/docs/self-hosted/deployment/docker#steps-to-deploy-chatwoot-using-docker-compose

**Reply by parsa-asgari** [reply]
I followed this guide:
https://www.chatwoot.com/docs/self-hosted/deployment/docker

It would be great if you could add that, if for some reason the user does not do the `docker compose up -d` immediately after the migration command, they will lose the chance at seeing the onboarding screen.

In my experience, I did a `docker compose down && docker compose up -d` and missed the oboarding screen. Afterwards, it went straight into login.

![image](https://github.com/chatwoot/chatwoot/assets/19937537/9b63598e-6df3-404d-b957-0825b6cf8a55)

**Reply by Gaspa79** [reply]
Had the same problem. @parsa-asgari is completely right about this.

A simple note "do not restart the containers/compose down or you'll miss the initial setup" would suffice

**Reply by vishnu-narayanan** [reply]
@parsa-asgari The Onboarding screen being shown is based on a REDIS key. The wrong order in which you run docker-compose commands should not affect it.

**Comment by Gaspa79** [comment]
Not about the order, but the fact that running the down command or doing
something else messes up the onboarding process.

Like I said: the guy who posted above me is right.

**Comment by sojan-official** [comment]
Updated documentation for this : https://deploy-preview-516--chatwoot-handbook.netlify.app/docs/self-hosted/faq#how-to-fix-the-onboarding-screen-issue-in-chatwoot
